integration/presto/src/main/java/org/apache/carbondata/presto/readers/BooleanStreamReader.java

```diff
@@ -86,4 +86,15 @@ public BooleanStreamReader(int batchSize, DataType dataType, Dictionary dictiona
     builder = type.createBlockBuilder(null, batchSize);
   }
 
+  @Override public void putObject(int rowId, Object value) {
```

Comment:
Already it has methods to handle put why added methods again in all readers. I don't think these changes required in all readers

Reply:
It is only required by stream reader. But I don't want to create many new adaptive objects, it will become complex.